base/libgit2/consts.jl

```diff
@@ -27,36 +27,38 @@ module Consts
     const REF_LISTALL  = REF_OID | REF_SYMBOLIC
 
     # checkout
-    const CHECKOUT_NONE                    = Cuint(0)
-    const CHECKOUT_SAFE                    = Cuint(1 << 0)
-    const CHECKOUT_FORCE                   = Cuint(1 << 1)
-    const CHECKOUT_RECREATE_MISSING        = Cuint(1 << 2)
-    const CHECKOUT_ALLOW_CONFLICTS         = Cuint(1 << 4)
-    const CHECKOUT_REMOVE_UNTRACKED        = Cuint(1 << 5)
-    const CHECKOUT_REMOVE_IGNORED          = Cuint(1 << 6)
-    const CHECKOUT_UPDATE_ONLY             = Cuint(1 << 7)
-    const CHECKOUT_DONT_UPDATE_INDEX       = Cuint(1 << 8)
-    const CHECKOUT_NO_REFRESH              = Cuint(1 << 9)
-    const CHECKOUT_SKIP_UNMERGED           = Cuint(1 << 10)
-    const CHECKOUT_USE_OURS                = Cuint(1 << 11)
-    const CHECKOUT_USE_THEIRS              = Cuint(1 << 12)
-    const CHECKOUT_DISABLE_PATHSPEC_MATCH  = Cuint(1 << 13)
-    const CHECKOUT_SKIP_LOCKED_DIRECTORIES = Cuint(1 << 18)
-    const CHECKOUT_DONT_OVERWRITE_IGNORED  = Cuint(1 << 19)
-    const CHECKOUT_CONFLICT_STYLE_MERGE    = Cuint(1 << 20)
-    const CHECKOUT_CONFLICT_STYLE_DIFF3    = Cuint(1 << 21)
-    const CHECKOUT_DONT_REMOVE_EXISTING    = Cuint(1 << 22)
+    @enumset(CHECKOUT,
+             CHECKOUT_SAFE                    = 1 << 0,
```

Comment:
Some of the ones further down are unnecessary, but part of me thinks it might be good practice to leave them in for reference to the headers.